pkg/notify/notifier/pushover/README.md

```diff
@@ -63,15 +63,30 @@ type PushoverOptions struct {
 #### Receiver CRD
 
 ```go
+// PushoverUserProfile includes userKey and other preferences
```

Comment:
Usually, we put implementation details as code comments, not as readme.
Chinese version is not necessary.

We'd better put pushover usages in the main readme.md like SMS, DingTalk and email etc. https://github.com/kubesphere/notification-manager/blob/master/README.md

Reply:
Moved. https://github.com/txfs19260817/notification-manager/blob/pushover/README.md#deploy-the-default-pushoverconfig-and-global-pushoverreceiver